integration-tests/src/test/resources/environment-configuration.yml

```diff
@@ -71,3 +71,7 @@ instanceEnv:
     ZWE_CACHING_SERVICE_PORT: 10016
     ZWE_CACHING_EVICTION_STRATEGY: reject
     ZWE_CACHING_STORAGE_SIZE: 100
+    EUREKA_CLIENT_REGISTRY-FETCH-INTERVAL-SECONDS: 5
+    EUREKA_INSTANCE_LEASE-EXPIRATION-DURATION-IN-SECONDS: 45
+    EUREKA_SERVER_LEASERENEWALINTERVALINSECONDS: 10
+    EUREKA_SERVER_RENEWAL-PERCENT-THRESHOLD: 0.85
```

Comment:
Would it be advantageous to disable self-preservation alltogether?

Reply:
we can try

Reply:
it seems to be bit slower if have any impact, I will revert to threshold